osf_tests/test_node.py

```diff
@@ -3505,7 +3504,6 @@ def test_queueing_on_node_updated(self, mock_request_update, node, user):
         task = handlers.get_task_from_queue('website.project.tasks.on_node_updated', predicate=lambda task: task.kwargs['node_id'] == node._id)
         assert 'contributors' in task.kwargs['saved_fields']
         assert 'node_license' in task.kwargs['saved_fields']
-        mock_request_update.assert_called_once()
```

Comment:
This would always provide a positive regardless of if `request_identifier_update` ran, so removed it as assert_called_once is no longer allowed in mock 2.0.0 .